frontend/package.json

```diff
@@ -6,7 +6,8 @@
     "ng": "ng",
     "start": "ng serve --proxy-config proxy.conf.json",
     "build": "ng build --prod",
-    "test": "ng test --single-run --code-coverage --no-progress --reporters dots",
+    "test": "ng test --single-run --code-coverage --no-progress --reporters dots,coverage-istanbul",
```

Comment:
coverage-istanbul is applied automatically when the --code-coverage option is passed, isn't it?